backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/bottomsheet/BpkBottomSheetState.kt

```diff
@@ -51,18 +51,6 @@ class BpkBottomSheetState internal constructor(
     internal val delegate: SheetState,
 ) {
 
-    constructor(
```

Comment:
we only ever use this with the remember function. there was a breaking change with a new density param, but as we're only using it with the remember function anyways might as well remove it instead